carbon-design-system/carbon-addons-iot-react
# yet another V3 upgrade

Closes #

**Summary**

- Next round of v3 upgrade pulling in all the changes of the last couple days. Again, just a formality. Everything has been approved, and the only merge conflicts were related to `onToggleAggregations` being added into the Table in V3.

**Change List (commits, features, bugs, etc)**

- merge conflicts in:
- `packages/react/src/components/Table/Table.jsx` - related to the addition of `onToggleAggregations` and removal of a comment
- `packages/react/src/components/Table/Table.story.jsx` - same as above
- `packages/react/src/utils/__tests__/__snapshots__/publicAPI.test.js.snap` - same as above

**Acceptance Test (how to verify the PR)**

- check and make sure everything looks okay at a glance

**Regression Test (how to make sure this PR doesn't break old functionality)**

- check and make sure everything looks okay at a glance. Again, should just be a formality. Everything is approved and no merge conflicts of note in this round.



**Things to look for during review**
